loops-and-arrays.js

```diff
@@ -3,35 +3,48 @@
 // Schrijf een script dat iedere naam in onderstaande array vervangt door een koosnaampje, door er -"je" achter te plakken.
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 namen zou bevatten!
 // ==========================================
+console.log("--opdracht 1--")
 
 const names = ["Henk", "Piet", "Fred", "Joop"];
 
+console.log(names)
+
+for (let i = 0; i < names.length; i++) {
+    names[i] = (names[i] + "je")
+}
+
+console.log(names)
 // Verwachtte uitkomsten:
 // Vóór het script zie je de originele waardes:
 // console.log(names) geeft: ["Henk", "Piet", "Fred", "Joop"]
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
 
-
-
 // ==========================================
 // Opdracht 2
 // Schrijf een script dat ieder EVEN getal in onderstaande array met 2 vermenigvuldigd, en ieder ONEVEN getal met 3
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 getallen zou bevatten!
 // ==========================================
+console.log("--opdracht 2--")
 
 const numbers = [2, 4, 5, 29, 38];
```

Comment:
Aan het einde van regel 31, kan je ";" ook weg laten, het is volgens aandachtspunten peer review "of alles of niets"